pkg/core/config/tenant.go

```diff
@@ -370,11 +413,11 @@ func (c *TenantConfiguration) AfterUnmarshal() {
 		if config.Expiry == 0 {
 			config.Expiry = 3600 // 1 hour
 		}
-		if config.Sender == "" {
-			config.Sender = "no-reply@skygear.io"
-		}
-		if config.Subject == "" {
-			config.Subject = "Verification instruction"
+		if config.EmailMessage.Subject() == "" {
+			if config.EmailMessage == nil {
+				config.EmailMessage = EmailMessageConfiguration{}
```

Comment:
Maybe we can use `default_zero_value:"true"` tag, so we don't need to check and assign zero value?

Reply:
So `default_zero_value` works with maps too? I didn't know that.